tests/test_dynamic_mock.js

```diff
@@ -31,15 +71,29 @@ test('one function returning the status code and body defines a full mock', func
     })
 
   request.get('http://example.test/def', function(err, resp, body) {
-    if (err) {
-      throw err
-    }
+    t.error(err)
     t.equal(resp.statusCode, 201)
     t.equal(body, 'DEF')
     t.end()
   })
 })
 
+test('one function returning an array can set headers', function(t) {
```

Comment:
This seems to duplicate the test "reply function returning array with status code, string body, and headers object" in `test_reply_function_sync.js`.

Reply:
yep. I removed this new one.